Chummer/Classes/clsOptions.cs

```diff
@@ -622,16 +633,16 @@ public static bool UseLogging
         /// <summary>
         /// Whether or not the app should use logging.
         /// </summary>
-        public static bool UseLoggingApplicationInsights
+        public static EnumUseLoggingApplicationInsights UseLoggingApplicationInsights
         {
-            get => _blnUseLoggingApplicationInsights;
+            get => (EnumUseLoggingApplicationInsights)Enum.Parse(typeof(EnumUseLoggingApplicationInsights), _enumUseLoggingApplicationInsights.ToString());
             set
             {
-                if (_blnUseLoggingApplicationInsights != value)
+                if (_enumUseLoggingApplicationInsights != value)
                 {
-                    _blnUseLoggingApplicationInsights = value;
+                    _enumUseLoggingApplicationInsights = value;
                     // Sets up logging if the option is changed during runtime
-                    if (value)
+                    if ((int)_enumUseLoggingApplicationInsights < 1)
```

Comment:
You can compare enums directly, so `if (_enumUseLoggingApplicationInsights < EnumUseLoggingApplicationInsights.onlymetric)` has same effect without any magic values of 1

Furthermore your if can be entirely removed and simplified to `TelemetryConfiguration.Active.DisableTelemetry = !(_enumUseLoggingApplicationInsights < EnumUseLoggingApplicationInsights.onlymetric)`
(Which, if I'm parsing it correctly, enables logging if `_enumUseLoggingApplicationInsights` one of `notset`, `onlylocal`, `onlymetric`, which I don't think is the intended behavior)